Human.cpp

```diff
@@ -0,0 +1,210 @@
+        crawl();
+        swim();
+        walk();
+        std::cout << std::endl;
+    }
+};
+
+class Mammals : public Amphibia {
+public:
+    std::string mainProduct{ "Milk" };
+
+    void getInfo() {
+        std::cout << "LIVING ORGANISM MAMMALS:" << "\n";
+        std::cout << "For living is nessary: " << gasForLife << ", " << substanceForLife << ", " << mainProduct << std::endl;
+        std::cout << "Structure of living organism: " << structure << std::endl;
+        std::cout << "Living organism have: " << supportАpparatus << ", " << lowerLimbs << std::endl;
+        crawl();
+        swim();
+        walk();
+        std::cout << std::endl;
+    }
+};
+
+class Primates : public Mammals {
+public:
+    std::string upperLimbs{ "hands" };
+   
+    void carry() {
+        std::cout << "This living organism can carry objects" << std::endl;
+    }
+
+    void getInfo() {
+        std::cout << "LIVING ORGANISM PRIMATES:" << "\n";
+        std::cout << "For living is nessary: " << gasForLife << ", " << substanceForLife << ", " << mainProduct << std::endl;
+        std::cout << "Structure of living organism: " << structure << std::endl;
+        std::cout << "Living organism have: " << supportАpparatus << ", " << lowerLimbs << ", " << upperLimbs << std::endl;
+        crawl();
+        swim();
+        walk();
+        carry();
+        std::cout << std::endl;
+    }
+};
+
+class Human : public Primates {
+public:
+    void toThink() {
+        std::cout << "This living organism can thinking" << std::endl;
+    }
+
+    void getInfo() {
+        std::cout << "LIVING ORGANISM HUMAN:" << "\n";
+        std::cout << "For living is nessary: " << gasForLife << ", " << substanceForLife << ", " << mainProduct << std::endl;
+        std::cout << "Structure of living organism: " << structure << std::endl;
+        std::cout << "Living organism have: " << supportАpparatus << ", " << lowerLimbs << ", " << upperLimbs << std::endl;
+        crawl();
+        swim();
+        walk();
+        carry();
+        toThink();
+        std::cout << std::endl;
+    }
+};
+
+
+int main()
+{
+    std::string userQuestion;
+    std::string::size_type n;
+    const int arraySize{ 7 };
+    std::array<std::string, arraySize> keyWords{ "bacterium", "invertebrat", "fish", "amphibia", "mammals", "primate", "human"};
+   
+    std::cout << "\n" << "Plaese click enter for start" << std::endl;
+    if (_getch() == 13)
+    {
+        std::cout << "Hello, please enter your qustion about living organism" << std::endl;
+    }
+    while (true)
+    {
+        std::getline(std::cin, userQuestion);
+        for (int i{}; i < arraySize; i++)
+        {
+            n = userQuestion.find(keyWords[i]);
+            if (n != std::string::npos) 
+            {
+                std::string findWord{ userQuestion.substr(n) };
+                if (findWord == "human")
+                {
+                    Human smartHuman;
+                    smartHuman.getInfo();
```

Comment:
Можно через временный объект
```suggestion
                    Human().getInfo();
```
Другой вариант, более правильный, использовать мапу, значение которой будет лямбда, и поиском искать нужную лямбду. Так получится избавится от кучи if